examples/spark2/src/main/java/org/apache/carbondata/benchmark/SDKReaderBenchmark.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.benchmark;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import org.apache.hadoop.conf.Configuration;
+
+import org.apache.carbondata.common.exceptions.sql.InvalidLoadOptionException;
+import org.apache.carbondata.core.constants.CarbonCommonConstants;
+import org.apache.carbondata.core.util.CarbonProperties;
+import org.apache.carbondata.sdk.file.*;
+
+/**
+ * Test SDK read performance
+ */
+public class SDKReaderBenchmark {
```

Comment:
This call not required. Please remove.

Reply:
we need know the SDK reader performance, and then we can know JNI performance lose by comparing with CSDK performance.  And @KanakaKumar told we should test SDK reader performance in 1.5.1. 
example has two benchmark currently， ConcurrentQueryBenchmark and SimpleQueryBenchmark。

If there are lack some test or  indepth (IO time, pages scanned and all) or granular， we can add.

Reply:
**you can have reader benchmarking code,** 

but it should just be a reader code that takes path and prints the rows and time taken.
But this example is extending data (writing data) That should not be there.  
Also for everything separate S3 example not required. current examples only set conf and it works. So don't add multiple examples also.